config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  resources :mechanics, only: [:show]
+  resources :mechanic_rides, only: [:new, :create]
+  resources :amusement_parks, only: [:show]
```

Comment:
Nice work making RESTful routes! Another RESTful option for your form (mechanic_rides) would be: 
```
resources :mechanics, only: :show do
    resources :rides, only: :create, controller: "mechanic_rides"
end
```

Reply:
Also you don't need to include `new` above because the form is rendered on the mechanic show page (i.e. there isn't a `new` page that renders that form separately